pyproject.toml

```diff
@@ -16,7 +16,8 @@ classifiers = [
 ]
 requires-python = ">=3.9"
 dependencies = [
-    "gramps[GUI,i18n]==5.2.*",
+    "gramps[GUI,i18n]==6.0.0b1",
```

Comment:
We can try with `"gramps[all] @ git+https://github.com/gramps-project/gramps.git@maintenance/gramps60"` now that the fixes have been merged into this branch.